src/shared/services/token-service.ts

```diff
@@ -0,0 +1,32 @@
+class TokenService {
+  private accessTokenKey: string;
+  private refreshTokenKey: string;
+
+  constructor() {
+    this.accessTokenKey = "access_token";
+    this.refreshTokenKey = "refresh_token";
+  }
+
+  getAccessToken(): string | null {
+    return localStorage.getItem(this.accessTokenKey);
+  }
+
+  setAccessToken(token: string): void {
+    localStorage.setItem(this.accessTokenKey, token);
+  }
+
+  getRefreshToken(): string | null {
+    return localStorage.getItem(this.refreshTokenKey);
+  }
+
+  setRefreshToken(token: string): void {
+    localStorage.setItem(this.refreshTokenKey, token);
+  }
+
+  removeTokens(): void {
+    localStorage.removeItem(this.accessTokenKey);
+    localStorage.removeItem(this.refreshTokenKey);
+  }
+}
+
+export default new TokenService();
```

Comment:
export default is not the best way to export files, replace to named export

Reply:
resolved